submissions/Kostiantyn_Rakhmanov/hooli-style-popup/index.html

```diff
@@ -0,0 +1,173 @@
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/tanslation.png" alt="#">
+                                        <p class="product__item-name">Translate</p>
+                                    </a>
+                                </div>
+
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/videos.png" alt="#">
+                                        <p class="product__item-name">Videos</p>
+                                    </a>
+                                </div>
+
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/search.png" alt="#">
+                                        <p class="product__item-name">Search</p>
+                                    </a>
+                                </div>
+
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/photos.png" alt="#">
+                                        <p class="product__item-name">Photos</p>
+                                    </a>
+                                </div>
+
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/apps.png" alt="#">
+                                        <p class="product__item-name">Apps</p>
+                                    </a>
+                                </div>
+
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/documents.png" alt="#">
+                                        <p class="product__item-name">Documents</p>
+                                    </a>
+                                </div>
+                            </div>
+                            <input class="products__checkbox-more" type="checkbox" id="products__checkbox-more">
+                            <label for="products__checkbox-more" class="products__more-label">More</label>
+                            <div class="products__more">
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/chat.png" alt="#">
```

Comment:
![image](https://user-images.githubusercontent.com/16196199/187038802-39c6f2ad-7b60-40c7-82b2-6839a7c30fb9.png)

Reply:
# - is not ok?
My friends told me that have to be, alt can't be - nothing. =)
Is it better to del all alt's with # inside?
Or just make them alt=""?

Reply:
No need to populate `alt` attribute for decorative images. The icon is a decorative image. 
If you remove `alt` attribute from the image it will be an invalid markup.
![image](https://user-images.githubusercontent.com/16196199/187039359-ece820b6-cef1-4641-a3ad-f78ef8d4dcdc.png)

Reply:
Thanks!